lib/slime/parser.ex

```diff
@@ -129,15 +138,19 @@ defmodule Slime.Parser do
   end
 
   defp parse_attributes(""), do: {"", []}
-  defp parse_attributes("(" <> line), do: parse_wrapped_attributes(line, ")")
-  defp parse_attributes("[" <> line), do: parse_wrapped_attributes(line, "]")
-  defp parse_attributes("{" <> line), do: parse_wrapped_attributes(line, "}")
+
   defp parse_attributes(line) do
-    match = @attr_group_regex |> Regex.run(line) |> List.first
-    offset = String.length(match)
-    {attrs, rem} = line |> String.split_at(offset)
-    attrs = parse_attributes(attrs, [])
-    {rem, attrs}
+    delim = String.first(line)
+    if Dict.has_key?(@attr_list_delims , delim) do
+      line = String.replace_prefix(line, delim, "")
```

Comment:
`String.replace_prefix/3` is a Elixir 1.2 feature,  so the current travis build is failing. Please fix it first.